mpi-proxy-split/README.md

```diff
@@ -67,6 +67,12 @@ See `man mana` or `nroff -man MANA_ROOT_DIR/manpages/mana.1` for the MANA man pa
 
    For convenience, add the MANA bin directory to your PATH.
  
+   ```bash
+   $ export PATH=$PATH:$MANA_ROOT/bin
+   ```
+   
+   Optionally, you can make this path persistent by doing:
+
    ```bash
    $ echo 'export PATH=$PATH:$MANA_ROOT/bin' >> ~/.bashrc
    $ source ~/.bashrc
```

Comment:
We don't want `source ~/.bashrc` here.  The idea is that we added to our path in the current shell, and optionally, we can add to the path in .bashrc for the sake of future shells.  So, there is no reason to do `source ~/.bashrc` in the current shell.  We've already extended the path  We don't need to do it again for this shell.